srcpkgs/rofi-wayland/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'rofi-wayland'
+pkgname=rofi-wayland
+version=1.7.0
+revision=1
+_ver=${version}+wayland1
+wrksrc=rofi-${_ver}
+build_style=meson
+configure_args="-Dxcb=disabled -Dcheck=disabled"
+hostmakedepends="ninja flex bison glib-devel pkg-config wayland-devel"
+makedepends="gdk-pixbuf-devel libxkbcommon-devel pango-devel
+ wayland-protocols wayland-devel"
+checkdepends="check-devel cppcheck xkeyboard-config"
+short_desc="Window switcher, run dialog and dmenu replacement"
```

Comment:
This should probably include something to differentiate this from the `rofi` package. Could just add "- wayland fork" or something...

Reply:
Correct, for short_desc